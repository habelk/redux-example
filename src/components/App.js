/**
 * Hello World app
 */
import React from 'react';

class App extends React.Component{
  render() {
    return (<div>React Testing</div>);
  }
}



export default App;
