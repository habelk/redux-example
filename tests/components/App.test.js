import React from "react";
import App from "../../src/components/App";


describe('App', () => {
    it('Test the text', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
    });
});