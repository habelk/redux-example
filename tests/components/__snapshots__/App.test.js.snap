// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App text 1`] = `
<div>
  React Testing
</div>
`;

exports[`App Test the text 1`] = `
<div>
  React Testing
</div>
`;
